src/DynamoCoreWpf/Controls/DynamoTextBox.cs

```diff
@@ -153,6 +153,11 @@ public bool Pending
         new public string Text
         {
             get { return base.Text; }
+            set
+            {
+                base.Text = value;
+                UpdateDataSource(true);
+            }
```

Comment:
@Benglin Then we need to rethink this design. I added a setter here so that I can do two-way data binding from the view.